### PR TITLE
Feat/come partecipare

### PR DESCRIPTION
**Pull request by Fladiir**
Added page come-partecipare

### CODE REVIEW

**Review by emnlpstr — APPROVED** [review]

File: src/layouts/CreaProfiloPage.jsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+import { announce } from '@react-aria/live-announcer';
+import faq from '../../contents/faq-page/faq.yml';
+import { SEO } from '../components/SEO';
+import seo from '../../contents/seo.yml';
+import content from '../../contents/crea-profilo/crea-profilo.yml';
+import { TimelineVerticalCards } from '../components/TimelineVerticalCards';
+import { HeroVideo } from '../components/HeroVideo';
+import { SideNavigationAccordion } from './sideNavigationAccordion';
+
+const { title: seoTitle, description: seoDescription } = seo.faqPage;
```
Comment (sebbalex) [inline src/layouts/CreaProfiloPage.jsx:16]: This is wrong here

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: src/layouts/CreaProfiloPage.jsx (hunk 1 above)
Comment (emnlpstr) [reply src/layouts/CreaProfiloPage.jsx]: fixed

File: src/components/hero/HeroHowToDo.jsx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+});
+
+export const HeroHowToDo = ({ title, body, image, imageMob }) => {
+  const classes = useStyles();
+
+  // const scrollIntoView = (id) => {
+  //   const item = document.querySelector('#' + id);
+  //   item.setAttribute('tabindex', '-1');
+  //   item.focus();
+  // };
```
Comment (sebbalex) [inline src/components/hero/HeroHowToDo.jsx:205]:
```suggestion

```

File: src/components/hero/HeroHowToDo.jsx [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+                                  className={classes.listItem}
+                                >
+                                  {item.item}
+                                </Link>
+                              </React.Fragment>
+                            ))}
+                          </div>
+                        </div>
+                      )) : ''}
+                    </div> */}
```
Comment (sebbalex) [inline src/components/hero/HeroHowToDo.jsx:248]: Remove please

File: src/layouts/CreaProfiloPage.jsx [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+
+  useEffect(() => {
+    setIsMobile(window.innerWidth < 992);
+    window.addEventListener('resize', () => {
+      setIsMobile(window.innerWidth < 992);
+    });
+    announce('Pagina caricata ' + faq.name);
+  }, []);
+
+  // const handleChange = (event) => {
```
Comment (sebbalex) [inline src/layouts/CreaProfiloPage.jsx:150]: same here

File: src/layouts/CreaProfiloPage.jsx [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+    }
+  }, [isMobile]);
+
+  return (
+    <>
+      <SEO title={seoTitle} description={seoDescription} />
+      <div className="sr-only">
+        <h2>{faq.name}</h2>
+      </div>
+      {/* <HeroSupport title={faq.hero.title} subtitle={faq.hero.subtitle} /> */}
```
Comment (sebbalex) [inline src/layouts/CreaProfiloPage.jsx:186]:
```suggestion

```

File: src/layouts/come-fare/KeypointSection.jsx [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+    <>
+      {lBlue ? (
+        <div className={classes.bgLightBlue}>
+          <Container
+            tag="section"
+            aria-labelledby="keypoints-section-title"
+            className={`${classes.keypointSection} pl-lg-2 pr-lg-2 pl-3 pr-3`}
+            /* className="pl-lg-2 pr-lg-2 pl-3 pr-3" */
+          >
+            {/* <Row>
```
Comment (sebbalex) [inline src/layouts/come-fare/KeypointSection.jsx:124]: same

File: src/layouts/come-fare/KeypointSection.jsx [hunk 7]
```diff
@@ -0,0 +1,359 @@
...
+          </Container>
+        </div>
+      ) : (
+        <Container
+          tag="section"
+          aria-labelledby={`keypoint-section-title-${number}`}
+          /* className="pl-lg-2 pr-lg-2 pl-3 pr-3" */
+          className={`${classes.keypointSection} pl-lg-2 pr-lg-2 pl-3 pr-3`}
+        >
+          {/* <Row>
```
Comment (sebbalex) [inline src/layouts/come-fare/KeypointSection.jsx:212]: same

File: src/layouts/come-fare/Keypoints.jsx [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+                </Col>
+              </Row>
+            </Col>
+          </Row>
+          <Row>
+            {list
+              ? list.map((listItem, index, array) => (
+                  <Col xs="12" md="6" lg="3" className={`${index + 1 === array.length ? 'mb-lg-0' : 'mb-5 mb-lg-0'}`}>
+                    <div className={classes.keypointNumber}>{listItem.number}.</div>
+                    {/* <img className={classes.keypointImage} src={`/assets/come-fare/${listItem.image}`} alt="" /> */}
```
Comment (sebbalex) [inline src/layouts/come-fare/Keypoints.jsx:150]: same

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: src/layouts/come-fare/Keypoints.jsx (hunk 8 above)
Comment (emnlpstr) [reply src/layouts/come-fare/Keypoints.jsx]: I removed all the comments

File: src/layouts/come-fare/KeypointSection.jsx [hunk 9]
```diff
@@ -0,0 +1,330 @@
...
+                </Col>
+              </Row>
+            )}
+          </Container>
+        </div>
+      ) : (
+        <Container
+          tag="section"
+          aria-labelledby={`keypoint-section-title-${number}`}
+          /* className="pl-lg-2 pr-lg-2 pl-3 pr-3" */
```
Comment (sebbalex) [inline src/layouts/come-fare/KeypointSection.jsx:194]:
```suggestion
```

**Review by sebbalex — APPROVED** [review]

**Pull request merged**